definition_tooling/validator/core.py

```diff
@@ -38,6 +38,10 @@ def validate_spec(spec: dict):
     if not spec.get("openapi", "").startswith("3"):
         raise err.UnsupportedVersion("Validator supports only OpenAPI 3.x specs")
 
+    for field in {"title", "description"}:
+        if not spec.get("info", {}).get(field):
+            raise err.MandatoryField(f"{field} is a required field")
```

Comment:
I'm a bit considering if the error message should possibly be more clear about these being expected in the "info" section or not. The check further down does at least say it's a required field for a POST route.